Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'acts_as_tree'
 gem 'cancancan', '~> 1.10'
 gem 'devise'
 gem 'figaro'
+gem 'thumbs_up', github: 'bouchard/thumbs_up', ref: 'f499a7'
```

Comment:
It's to resolve an issue about the number of args. For reference: https://github.com/bouchard/thumbs_up/issues/96